pkg/grpcfactory/client.go

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
Could we just do a `LICENSE` file? (nitpick)

Reply:
yes we can, will remove from all in one go